src/components/modals/options/AnimationOptionsModal.vue

```diff
@@ -25,7 +25,10 @@ export default {
       reality: false,
       background: false,
       blobSnowflakes: 16,
-      isS11Active: false
```

Comment:
`isS11Active` is still being used on line 142; this doesn't cause any issues if it's removed, but keeping every used data() prop explicitly named is preferred